trieste/models/keras/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+This package contains the primary interface for deep neural network models. It also contains a
+number of :class:`TrainableProbabilisticModel` wrappers for Keras-based models. Note that currently
+copying/saving models is not supported, so in a Bayes Opt loop `track_state` should be set False.
+Note as well that `tf.keras.backend.set_floatx()` should be used to set the desired float type.
```

Comment:
Is it important to be able to set the float type? This isnt something we offer for gpflow models etc

Reply:
Keras is using float32 by default, while trieste is on float64 - so far I haven't encountered issues as Sebastian did in GPflux, but I thought it would be prudent to instruct Keras to set it to use float64

perhaps I should make it a recommendation, as there are no errors as far as I can see

Reply:
Sounds good